src/modules/fundingManager/bondingCurveFundingManager/ParibuChanges/ERC20Issuance.sol

```diff
@@ -0,0 +1,93 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity 0.8.23;
+
+// External Interfaces
+import {IERC20Issuance} from "./IERC20Issuance.sol";
+
+// External Dependencies
+import {ERC20Upgradeable} from "@oz-up/token/ERC20/ERC20Upgradeable.sol";
+import {ContextUpgradeable} from "@oz-up/utils/ContextUpgradeable.sol";
+import {OwnableUpgradeable} from "@oz-up/access/OwnableUpgradeable.sol";
+
+// External Libraries
+import {SafeERC20} from "@oz/token/ERC20/utils/SafeERC20.sol";
+
+/// @title ERC20Issuance Token
+/// @author Inverter Network.
+/// @notice This contract creates an ERC20 token with the ability to mint and burn tokens and a supply cap.
+/// @dev The contract implements functionalties for:
+///         - Whitelisted mint and burn
+///         - A supply cap
+///         - Flexible decimals in initialization
+contract ERC20Issuance is
+    IERC20Issuance,
+    ERC20Upgradeable,
```

Comment:
Has this question been answered already or is this still an open question?

Reply:
In any case I tend to agree with @Zitzak  though, as upgradeable tokens are always a very delicate topic. If there is no really good reason to have it like this, I think it shouldn't be upgradeable?

Reply:
We kind of agreed on leaving it upgradeable just in case we need to do some extra changes. The only thing I can think of right now is something like replacing minter address with minter whitelist, but maybe something else comes up...